Cargo.toml

```diff
@@ -124,6 +124,7 @@ greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", r
 humantime = "2.1"
 humantime-serde = "1.1"
 itertools = "0.10"
+jsonb = { git = "https://github.com/CookiePieWw/jsonb.git", branch = "trigger-preserve-order-feature" }
```

Comment:
Is it a temporary repository?

Reply:
Yes it's temporary. `jsonb` conflicts with greptimedb for `serde_json`. I've open an issue for it (https://github.com/datafuselabs/jsonb/issues/55), if the change works for them then we can use their repo, or I may find  other repo for jsonb format.